ui/src/components/login/LoginComponents.tsx

```diff
@@ -24,7 +25,7 @@ export const LoginContainer = ({ children }: { children: React.ReactNode }) => (
             display: "flex",
             flexDirection: "row",
             width: "800px",
-            height: "500px",
+            height: "550px",
```

Comment:
width and height will have to be update to have dynamic width and heigth, to make website responsive to different sizes of the page. Things to would be to use function of scss calc()